docs/requirements.txt

```
@@ -22,24 +22,24 @@
 # struct - builtin - PSF license
 # argparse - builtin - PSF license
 # setuptools - MIT license
-setuptools>=38.0, !=71.0.1, !=72.0, !=69.4.0, !=69.3.0, !=60.3.0, !=59.1
+setuptools>=75.0
 # virtualenv - MIT license
 #virtualenv>=15.0.1
 # pgpy - BSD 3-Clause licensed
 #pgpy>=0.4.1
 # tox - MIT license
 #tox>=3.0.0, !=3.16.1
 # wheel - MIT license
-wheel>=0.37.0
+wheel>=0.44
 # pip - MIT license
-pip>=19.0
+pip>=22.0
 # build - MIT license
-# build>=0.5.1, !=1.0.3, !=0.6, !=0.6.1 !=1.1.0 !=1.2.0
+build>=1.1.1, !=1.2.0
```

as discussed in https://github.com/reactive-firewall/multicast/pull/148#discussion_r1785754889

broken versions:
1.2.2.post1
1.2.0

python3.9 supported for 1.2.1+ so can version bump for consistancy

resulting recommendation:
```suggestion
build>=1.2.1, !=1.2.2.post1
```